pom.xml

```diff
@@ -321,7 +321,7 @@
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
-        <version>2.1</version>
+        <version>2.4</version>
```

Comment:
We have had some tough luck with such version increases which have pulled in conflicting transitive dependencies. Making a note here, so we can watch this one.